script/sync-dod-certs

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# script/sync-dod-certs: update the CA bundle with DOD intermediate and root CAs
+
+echo "Resetting CA bundle..."
+rm ssl/server-certs/ca-chain.pem || true
+touch ssl/server-certs/ca-chain.pem
+
+# only for testing
+echo "Copy in testing client CA..."
+cat ssl/client-certs/client-ca.crt >> ssl/server-certs/ca-chain.pem
+
+# dod intermediate certs
+echo "Adding DoD intermediate certs"
+curl -s https://militarycac.com/maccerts/AllCerts.p7b | openssl pkcs7 -print_certs -inform der -outform pem >> ssl/server-certs/ca-chain.pem
+echo "Adding DoD root certs"
+curl -s https://militarycac.com/maccerts/RootCert2.cer | openssl x509 -inform der -outform pem >> ssl/server-certs/ca-chain.pem
```

Comment:
@ddsdevon I'm using militarycac.com for this for now, but it would be good to find a more official source. The [DISA PKI site](https://crl.chamb.disa.mil/) doesn't let you download root CAs directly. (And it doesn't have a convenient bundle of the intermediates....) Do you know a better source?

Reply:
militarycac is what everyone uses. Yes, it looks awful, but without being on NIPR I don't know a better source for you.

Reply:
Fake news, NOONE should be using militarycac.com... there are .mil sources for these, I'll dig it up.

Reply:
@dandds https://iasecontent.disa.mil/pki-pke/Certificates_PKCS7_v5.3_DoD.zip
They are .p7b files unfortunately, but it can be converted readily enough:
`openssl pkcs7 -print_certs -in CERTFILE.p7b -out OUTFILE.pem`

Reply:
Okay, thanks. I'll work on that change.

Reply:
@ddsdevon The README for that package has instructions for validating the contents, which seems like a good thing for our script to do. It mentions checking the fingerprint of  `DoD_PKE_CA_chain.pem`, which isn't in the bundle. It's in [previous bundle versions](http://iasecontent.disa.mil/pki-pke/Certificates_PKCS7_v5.0u1_DoD.zip), though. Do you think validation is worth pursuing, and is there a better way to validate the bundle?

Reply:
For now, it probably shouldn't be required if they are inconsistent about whether they include the file or not.
Maybe a "if DoD_PKE_CA_chain.pem present, then validate" conditional?